### ISSUE TITLE
File and CloudinitConfig resources are deprecated

### ISSUE THREAD

**Issue by Frassle** [pulumi/pulumi-terraform-template#3]
The File and CloudinitConfig Resources are deprecated. They are legacy shims from before terraform supported the concept of Data Sources.

I'd PR to just remove them but then tfbridge emits warnings that Resources are missing. Is there a way to tell tfbridge to explicitly ignore resources?

https://github.com/pulumi/pulumi-terraform-template/blob/adc4bf573a2363ffd9d65275b6904b701d7a11bc/resources.go#L75

**Comment by pgavlin** [comment]
> I'd PR to just remove them but then tfbridge emits warnings that Resources are missing. Is there a way to tell tfbridge to explicitly ignore resources?

I don't believe that there is, no. I've filed https://github.com/pulumi/pulumi-terraform/issues/233 to track that feature.
